src/graphics/generic_button.c

```diff
@@ -26,6 +28,17 @@ static int get_button_min(const mouse* m, int x, int y, generic_button* buttons,
     return 0;
 }
 
+void generic_button_border_draw(const generic_button *button, int has_focus)
+{
+    button_border_draw(button->x, button->y, button->width, button->height, has_focus);
+}
+
+void generic_button_draw_text_centered(int group, int number, const generic_button *button, font_t font)
+{
+    int y = button->y + button->height / 2 - 5;
+    lang_text_draw_centered(group, number, button->x, y, button->width, FONT_NORMAL_BLACK);
+}
+
```

Comment:
This may be nice for the future, but right now I'd just keep the border code separate as it was.

Besides, if we're "widgeting" the generic button in the future, we might as well provide a `text` field to the structure and then, when calling `generic_button_draw`, it would automatically draw the text. But that's something for the future, not right now.

Reply:
I’ll take a look tonight but I may not be able to bring myself to change it back. Running after all those hardcoded constants in the code was just awful, and these are some pretty tame functions. I’d rather add the text field to the structure and keep moving forwards.

Reply:
I can do it, just set the x and y to some value and I'll fix.